src/domain/fastapiactuatorserver.py

```diff
@@ -0,0 +1,48 @@
+"""FastAPI Actuator server module"""
+
+import logging
+import threading
+import uvicorn
+
+logger = logging.getLogger(__name__)
+
+
+class FastAPIActuatorServer:
+    def __init__(self, ip_address: str, port: int, certificate: str, key: str):
+        self.ip = ip_address
+        self.port = port
+        self.certificate = certificate
+        self.key = key
+
+    def run(self):
+        """Method to run the FastAPI Actuator server with SSL in a separate thread."""
+        config = uvicorn.Config(
+            app="actuatorserver_app:app",
+            host=self.ip,
+            port=self.port,
+            ssl_certfile=self.certificate,
+            ssl_keyfile=self.key,
+        )
+        server = uvicorn.Server(config)
+
+        thread = threading.Thread(target=server.run)
+        if thread:
+            thread.start()
```

Comment:
How do we stop this thread? remember to add stop() method or a variable that when set allows to exit the run() loop